inc/error_msg.h

```diff
@@ -0,0 +1,100 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   error_msg.h                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/11/20 11:44:43 by gwolf             #+#    #+#             */
+/*   Updated: 2023/11/20 11:49:54 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef ERROR_MSG
+# define ERROR_MSG
+
+/* ====== LIBRARIES ====== */
+
+// Standard C libs
+# include <stdio.h>
+# include <errno.h>
+
+// Home-grown libs
+# include "ft_fd.h"
+# include "entities.h"
+
+/* ====== FUNCTIONS ====== */
+
+// error_msg_generic.c
+
+/**
+ * @brief Print generic error message.
+ *
+ * Prints "Error\n" and then
+ * - prints the passed error msg with perror().
+ * - if errno = 0 just the msg gets printed with ft_putendl_fd()
+ * @param msg			Error message to print.
+ * @param save_errno	Errno at the time of calling the function.
+ */
+void	ft_perror(char *msg, int save_errno);
+
+/**
+ * @brief Print error message with a number
+ *
+ * The number gets enclosed with <>. *
+ * @param msg		Error message to print.
+ * @param number	Number to print.
+ */
+void	ft_perror_number(char *msg, int number);
+
+// error_msg_check.c
+
+/**
+ * @brief Error message if entity count is not correct.
+ *
+ * @param type Entity type.
+ * @param max Maximum number for entity type.
+ * @param count Found number for entity type.
+ * @param high Controls if count is too high or too low.
+ */
+void		ft_perror_count(t_type type, int max, int count, bool high);
+
+/**
+ * @brief Error message if converted number is out of range.
+ *
+ * @param line Start position of converted number.
+ * @param offset Offset from start position.
+ * @param min Minimum for number.
+ * @param max Maximum for number.
+ * @return false Always returns false.
+ */
+bool		ft_perror_range(char *line, size_t offset, int min, int max);
```

Comment:
Because it can then be used in function return chains, i.e. return (ft_perror_range) - in case of error. 
Updated doxy comment: "Always returns false to chain msg and false in return."